charts/dis/values-prod.yaml

```diff
@@ -14,6 +14,7 @@ env:
   SYNTHETIC_NFT_ADDRESS: '0x4804e8D1661cd1a1e5dDdE1ff458A7f878c0aC6D'
   TOKEN_EXCHANGE_KEY_SET_URL: 'https://auth.dimo.zone/dex/keys'
   TOKEN_EXCHANGE_ISSUER: 'https://auth.dimo.zone/dex'
+  RPC_URL: https://polygon-rpc.com
```

Comment:
we might need to add a private rpc via a secret so we don't get rate limited

Reply:
yep agreed, wanted to populate this with something in the meantime